### PR TITLE
Cassandra connector docs

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

File: cassandra/src/test/java/akka/stream/contrib/cassandra/CassandraSourceTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * All the tests must be run with a local Cassandra running on default port 9042.
+ */
```
Comment (johanandren) [inline cassandra/src/test/java/akka/stream/contrib/cassandra/CassandraSourceTest.java]: 👍

File: cassandra/src/test/java/akka/stream/contrib/cassandra/CassandraSourceTest.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+  @Test
+  public void streamStatementResult() throws Exception {
+    //#statement
+    final Statement stmt = new SimpleStatement("SELECT * FROM akka_stream_java_test.test").setFetchSize(20);
+    //#statement
+
+    //#run-source
+    final CompletionStage<List<Row>> rows = CassandraSource.create(stmt, session)
+      .runWith(Sink.seq(), materializer);
```
Comment (johanandren) [inline cassandra/src/test/java/akka/stream/contrib/cassandra/CassandraSourceTest.java]: Heh, if all you want is to get cassandra rows into a `List` then maybe using streams is a bit of overkill? :D  Would it be possible to have a sample that is a little bit more realistic (pretend data wouldn't fit in memory or such)?

File: docs/src/main/paradox/cassandra.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+# Cassandra Connector
+
+The Cassandra connector provides a way to transform a Cassandra's ResultSet into a Akka Stream's Source.
```
Comment (johanandren) [inline docs/src/main/paradox/cassandra.md]: "The Cassandra connector provides a way to provide the result of a Cassandra query as a stream of rows", currently it sounds like you fetch the results into a `ResultSet` from the database and then stream rows from that, but the whole point is bounded memory, no?

File: docs/src/main/paradox/cassandra.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+And finally create the source using any method from the @scaladoc[CassandraSource](akka.stream.contrib.cassandra.CassandraSource$) factory and run it.
+
+Scala
+: @@snip (../../../../cassandra/src/test/scala/akka/stream/contrib/cassandra/CassandraSourceSpec.scala) { #run-source }
+
+Java
+: @@snip (../../../../cassandra/src/test/java/akka/stream/contrib/cassandra/CassandraSourceTest.java) { #run-source }
+
+Here we used a basic sink to complete the stream by collecting all of the stream elements to a collection. The power of streams comes from building larger data pipelines which leverage backpressure to ensure efficient flow control. Feel free to edit the example code and build @extref[more advanced stream topologies](akka-docs:scala/stream/stream-introduction).
```
Comment (johanandren) [inline docs/src/main/paradox/cassandra.md]: Ah, ok, maybe it is ok then with such a simplistic sample given this note.

File: cassandra/src/test/java/akka/stream/contrib/cassandra/CassandraSourceTest.java (hunk 2 above)
Comment (ktoso) [inline cassandra/src/test/java/akka/stream/contrib/cassandra/CassandraSourceTest.java]: would be nice yeah

Comment (2m) [inline cassandra/src/test/java/akka/stream/contrib/cassandra/CassandraSourceTest.java]: Agree that something more comprehensive would be nice. But maybe a separate guide that would use these connectors in a concert would be more beneficial. Could be a blog post in Akka blog.

**Pull request merged**